plugin/builtin/patchtransformer/PatchTransformer_test.go

```diff
@@ -746,8 +743,72 @@ spec:
       - image: test-image
         name: test-deployment
         ports:
+        - containerPort: 8080
+          name: take-over-the-world
+          protocol: TCP
         - containerPort: 8080
           name: disappearing-act
           protocol: TCP
 `)
 }
+
+func TestPatchTransformerSecondaryMergeKey(t *testing.T) {
+	th := kusttest_test.MakeEnhancedHarness(t).
+		PrepBuiltin("PatchTransformer")
+	defer th.Reset()
+
+	th.RunTransformerAndCheckResult(`
+apiVersion: builtin
+kind: PatchTransformer
+metadata:
+  name: test-transformer
+patch: |-
+  apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: test-transformer
+    labels:
+      test-transformer: did-my-job
+  spec:
+    template:
+      spec:
+        containers:
+        - image: test-image
+          name: test-deployment
+          ports:
+          - containerPort: 8080
+            protocol: different
```

Comment:
maybe use a valid protocol, this is sort of confusing :-) UDP?

Reply:
will do, thank you